<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadMeshDialog</class>
 <widget class="QDialog" name="LoadMeshDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>317</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>317</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>317</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="meshEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid  rgb(209, 209, 209);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="texEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid  rgb(209, 209, 209);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadMesh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgba(155, 155, 155,0);
border-radius:10px;
border: 2px solid  rgb(223, 223, 223);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
background-color: rgb(223, 223, 223);
border: 2px solid  rgb(223, 223, 223);
color: rgb(115, 115, 115);
}
</string>
   </property>
   <property name="text">
    <string>Load Mesh</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTexture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgba(155, 155, 155,0);
border-radius:10px;
border: 2px solid  rgb(223, 223, 223);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
background-color: rgb(223, 223, 223);
border: 2px solid  rgb(223, 223, 223);
color: rgb(115, 115, 115);
}
</string>
   </property>
   <property name="text">
    <string>Load Texture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Texture (Optional)</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgba(155, 155, 155,0);
border-radius:10px;
border: 2px solid  rgb(223, 223, 223);
color: rgb(255, 255, 255);
padding:5px;
width:50px;
}
QPushButton:hover
{
background-color: rgb(223, 223, 223);
border: 2px solid  rgb(223, 223, 223);
color: rgb(115, 115, 115);
}
</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
